app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -153,15 +196,26 @@ private class HomeTileAdapter(
             if (hasFocus) {
                 backgroundResource = R.drawable.home_tile_title_focused_background
                 textColor = Color.WHITE
+                menuButton.visibility = View.VISIBLE
             } else {
                 backgroundResource = 0
                 textColor = Color.BLACK
+                menuButton.visibility = View.GONE
             }
             titleView.setBackgroundResource(backgroundResource)
             titleView.setTextColor(textColor)
         }
     }
 
+    fun getItemAtPosition(position: Int): HomeTile {
+        return tiles[position]
```

Comment:
Very dangerous! Since this is a public function, what if the position passed in is out of the range? You should check the index and return null if it's out of range. Also document that behavior in a javadoc style comment like /**